src/chrome/content/rules/DN.dk.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+	Non-functional domains:
+
+		- fordel ¹
+		- historie ¹
+		- tur ¹
+
+	¹: Refused
+-->
+<ruleset name="DN.dk (partial)">
+	<target host="dn.dk" />
+	<target host="www.dn.dk" />
+
+	<exclusion pattern="^http://www\.dn\.dk/([dD]efault\.aspx|\?).*" />
```

Comment:
I didn't come across any parts of the site where that would be a problem, at least.